benchmark_runner/common/template_operations/templates/vdbench/vdbench_data_template.yaml

```diff
@@ -22,10 +22,10 @@ template_data:
       DIRECTORIES: 600
       FILES_PER_DIRECTORY: 10
       SIZE_PER_FILE: 10
-      limits_cpu: 20m
+      limits_cpu: 2
       limits_memory: 2Gi
-      requests_cpu: 2
-      requests_memory: 2Gi 
+      requests_cpu: 500m
```

Comment:
For perf_ci, both 'limits_cpu' and 'requests_cpu' should be equal to the vcpu total (set to 60 below)

Reply:
I have updates it accordingly:
```
      requests_cpu: 500m
      requests_memory: 512Mi
```